cookbooks/longitudinal_examples.md

```diff
@@ -170,6 +170,22 @@ WITH samples AS
 SELECT approx_distinct(client_id) FROM samples WHERE max_concurrent_tabs > 100
 ```
 
+#### Keyed scalars
+
+Retrieve all the keys for a given scalar and sum all values for each key giving
+one row per key:
```

Comment:
Should we remove the channel groupby and filter as it is incidental to this recipe?

I think it's pretty neat (I've never seen that interactive filter before!) so we could alternatively add it to the description.